SQL/2016-03-10-DefaultMRIUploadPath.sql

```diff
@@ -0,0 +1 @@
+UPDATE Config AS c, ConfigSettings AS cs SET c.value="/data/incoming/" WHERE c.ConfigID=cs.ID AND cs.Name="MRIUploadIncomingPath";
```

Comment:
This should include add a where clause to only update if it's set to the default, and not overwrite existing variables. (Patch only)